.eslintrc.json

```diff
@@ -36,6 +36,38 @@
         "args": "after-used",
         "argsIgnorePattern": "^_"
       }
-    ]
+    ],
+    "import/order": [
+      "error",
+      {
+        "pathGroups": [
+          {
+            "pattern": "{react,jotai,jotai/*}",
+            "group": "external",
+            "position": "before"
+          },
+          {
+            "pattern": "modules/**",
+            "group": "builtin",
+            "position": "after"
```

Comment:
doesn't it make sense to use before internal too? (so first modules, then for example api,abis, etc.)

I guess we can define many different groups "before internal" and i would think they will respect the order of definition (didn't try). If this is the case, it feels easier to read, if we just create the groups we want like that